package.json

```diff
@@ -107,8 +101,18 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Enable/disable Unibeautify formatter"
+                },
+                "unibeautify.defaultConfig": {
+                    "type": "string",
+                    "default": null,
+                    "description": "Type in a path to your .unibeautifyrc outside the workspace folder. This will override the projects .unibeautifyrc config file. Note: The file path must be absolute for a directory or file!"
+                },
+                "unibeautify.extendConfig": {
```

Comment:
Part of me feels like supporting `extendConfig` is something which can come later. I support allowing users to set `defaultConfig` such that they can have a file always to fallback on for beautification settings.
However, once we start incorporating merging of multiple `.unibeautifyrc` files, it's another thing to worry about and I'm not comfortable merging yet.

Features are easy to add and hard to remove 😜 .

@muuvmuuv could you remove `extendConfig` from this PR for now? I'd like to get `defaultConfig` support merged in ASAP and I want to keep this Pull Request laser focused. Thanks.

cc @szeck87 thoughts?